<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigSelect</class>
 <widget class="QWidget" name="DigSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>left Half:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Col 0-149</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>right Half:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Col 150-299</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selAll">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>340</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Select All Pixels</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selNone">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>380</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear All Pixels</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>470</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept Selection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="discardButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>510</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Discard Selection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selLeft">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>250</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Left Pixels</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selRight">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>290</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Right Pixels</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
